src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -124,20 +124,20 @@ public string SubmitTextOverride
 
         GetSuggestionsCommand = new DelegateCommand(OnGetSuggestionsCommandExecute, null, threadingContext.JoinableTaskFactory);
 
-        var getSuggestionsAutomatically = _globalOptionService.GetOption(InlineRenameUIOptionsStorage.GetSuggestionsAutomatically);
-        IsUsingResultPanel = getSuggestionsAutomatically;
-        IsUsingDropdown = !IsUsingResultPanel;
         SetupTelemetry();
-        if (IsUsingResultPanel && IsSuggestionsPanelExpanded)
+
+        this.SupportsAutomaticSuggestions = !_globalOptionService.GetOption(InlineRenameUIOptionsStorage.GetSuggestionsAutomatically);
```

Comment:
i don't understand the `!` here.  The negation of things is *really* confusing.  If hte option is "GetSuggestionsAutomatically" and the user has that set to true... why are we setting SupportsAutomaticSuggestions to false.  This is very confusing.